### DISCUSSION TITLE
Implemеnt any DB adapter before release

### DISCUSSION THREAD

**Discussion by vjik** [yiisoft/rbac#109] [Ideas]
In process of creation a DB adapter architectural problems may appear. 
We can make `yiisoft/rbac-db` or `yiisoft/rbac-spiral-db`.

**Comment by samdark** [comment]
Yes. That is a good idea.

**Comment by rustamwin** [comment]
There is already such packages: https://github.com/yiisoft/rbac-db, https://github.com/yiisoft/rbac-spiral-db

**Reply by rustamwin** [reply]
But rbac-spiral-db should be renamed to rbac-cycle-db

**Reply by vjik** [reply]
> There is already such packages: https://github.com/yiisoft/rbac-db, https://github.com/yiisoft/rbac-spiral-db

They are without implementation. After refactoring of `yiisoft/rbac` need implements one of this packages.

**Reply by samdark** [reply]
@rustamwin renaming done.

**Comment by vjik** [comment]
Lock in favor https://github.com/yiisoft/rbac/issues/112
